efs.tf

```
@@ -0,0 +1,41 @@
+locals {
+  efs_name = "${var.name}-efs"
+
+  efs_tags = merge(
+    var.tags,
+    {
+      "Name" = local.efs_name
+    },
+  )
+}
+
+resource "aws_security_group" "efs" {
+  name_prefix = local.efs_name
+  description = "Attached to the EFS mount points"
+  tags        = local.efs_tags
+  vpc_id      = var.vpc_id
+
+  ingress {
+    description     = "Allow traffic from instances and other specified groups"
+    from_port       = 2049
+    protocol        = "TCP"
+    security_groups = concat([aws_security_group.this.id], var.efs_additional_allowed_security_groups)
+    to_port         = 2049
```

If there's more than one item, split it into multiple lines
```suggestion
    description = "Allow traffic from instances and other specified groups"
    from_port   = 2049
    protocol    = "TCP"
    to_port     = 2049
    
    security_groups = concat(
      [aws_security_group.this.id],
      var.efs_additional_allowed_security_groups
    )
```